scss/snowdrop.scss

```diff
@@ -7,7 +7,7 @@ $yellow:  #ffc72a;   //   ""   yellow
 $blue:    #69a3b9;   //   ""   blue
 
 $primary: $red;
-$info:    $blue;
+$info:    #0074c1;
```

Comment:
I may have misunderstood your question before. I don't like the 'semantic' color variables being set to a literal hex value. The purpose of defining a color palette with named colors (blue, red, etc.) is the semantic color variables can reference named colors from the palette. That way we can say "info is blue" without specifying exactly which blue. If we decide to change to a different blue later, all of the blue things will still be the same color. Otherwise, you end up with things like 
![image](https://user-images.githubusercontent.com/2037405/195927763-6cf4240f-d905-40eb-8406-117e86cca71b.png)
where you have two slightly different shades of blue adjacent to one another.